bot/extensions/custom_roles/commands.py

```diff
@@ -105,6 +105,16 @@ async def myrole(
             ephemeral=True,
         )
 
+    @app_commands.command(name="set_log_channel")
```

Comment:
The name is too generic, set_log_channel -> for what?
We also have several logs channels that are going to go in the guild config

Need to find a good solution for the increasing number of configuration variables.